### ISSUE TITLE
Daily "Scan media folders" not executing

### ISSUE THREAD

**Issue by dsiminiuk** [airsonic/airsonic#1344]
## Problem description
I have a shell script that downloads mp3 files daily at 21:30. Until recently the "Scan Media Folders" set to 23:00 daily was working but now it is not.

### Steps to reproduce
1. Download new files into the media path.
2. Wait for the Daily Scan
3. Observe no new files in the respective path.

## System information
Airsonic version: 10.4.1-RELEASE
Operating system: CentOS Linux 7
Java version: OpenJDK Runtime Environment (build 1.8.0_222-b10)
Proxy server: None
Client: Chrome, but out of scope for this issue
Language: English

## Additional notes
The new files are found in the library when a manual scan is executed, it is only the automatic scan that does not appear to be running or finding new media. Server time is correct and synced with NTP.

**Comment by jvoisin** [comment]
Do you have anything weird in your logs?

**Comment by dsiminiuk** [comment]
Nothing I can say is troublesome.

```Oct 17 23:00:04 airsonic java: 2019-10-17 23:00:04.797  INFO --- o.a.p.s.MediaScannerService              : Scanned media library with 250 entries.
Oct 17 23:00:05 airsonic java: 2019-10-17 23:00:05.483  INFO --- o.a.p.s.MediaScannerService              : Scanned media library with 500 entries.
snip
Oct 17 23:03:33 airsonic java: 2019-10-17 23:03:33.971  INFO --- o.a.p.s.MediaScannerService              : Scanned media library with 42345 entries.
Oct 17 23:03:33 airsonic java: 2019-10-17 23:03:33.971  INFO --- o.a.p.s.MediaScannerService              : Marking non-present files.
Oct 17 23:03:33 airsonic java: 2019-10-17 23:03:33.997  INFO --- o.a.p.s.MediaScannerService              : Marking non-present artists.
Oct 17 23:03:33 airsonic java: 2019-10-17 23:03:33.998  INFO --- o.a.p.s.MediaScannerService              : Marking non-present albums.
Oct 17 23:03:34 airsonic java: 2019-10-17 23:03:34.017  INFO --- o.a.p.s.MediaScannerService              : Completed media library scan.
Oct 17 23:03:34 airsonic java: 2019-10-17 23:03:34.282  INFO --- o.a.p.service.PlaylistService            : Starting playlist import.
Oct 17 23:03:34 airsonic java: 2019-10-17 23:03:34.282  INFO --- o.a.p.service.PlaylistService            : Completed playlist import.
```

**Comment by fxthomas** [comment]
Can you try to run `touch /path/to/files` right before a scan happens? I don't know how files are downloaded, but it's possible that the last-modified date is not updated for a directory between the root of your media folder and the directory in which you download your files, causing Airsonic to ignore it.

**Comment by dsiminiuk** [comment]
I'll let the process run its course tonight and check the media scan then check the timestamps and post an update.

I moved the download script from the original server (not the airsonic VM) to the airsonic VM today.  I doubt moving the script will matter. I am saving the files in the same place on a mounted CIFS share. I moved the script it just so everything related to airsonic is on the same machine.
